pkg/cmd/run/watch/watch.go

```diff
@@ -73,6 +74,7 @@ func NewCmdWatch(f *cmdutil.Factory, runF func(*WatchOptions) error) *cobra.Comm
 	}
 	cmd.Flags().BoolVar(&opts.ExitStatus, "exit-status", false, "Exit with non-zero status if run fails")
 	cmd.Flags().IntVarP(&opts.Interval, "interval", "i", defaultInterval, "Refresh interval in seconds")
+	cmd.Flags().Uint64VarP(&opts.Attempt, "attempt", "a", 0, "The attempt number of the workflow run")
```

Comment:
We will want to revert these changes as `gh run watch` is really only intended for the latest attempt.

Reply:
Reverted in 69cff80ba9d3d38b773c9360d525cbd72ed00a9e.